### DISCUSSION TITLE
Proyectos grandes

### DISCUSSION THREAD

**Discussion by luzmoreno03** [hackatomik/SISCOx256#49] [Git y GitHub | Lab]
En proyectos grandes, ¿cómo se asignan permisos y acceso a los repositorios para que los desarrolladores puedan obtener el acceso que necesitan sin comprometer la seguridad del código?

**Comment by quantumquirkxyz** [comment]
En proyectos grandes, se asignan permisos a los miembros del equipo para asegurar que cada uno pueda hacer lo que necesita sin comprometer la seguridad del código. Por ejemplo, algunos pueden tener acceso completo, mientras que otros solo pueden ver el código o hacer comentarios. Se pueden configurar permisos para diferentes partes del proyecto y revisar cambios antes de que se integren al código principal. También se lleva un registro de quién hace cada cambio para mantener el proyecto seguro y bien organizado.

Aunque, considero que @JamSarmientoH9 es quien te puede orientar más con esta pregunta.

 Jamkeatesis Sarmiento | +507 6585-2329

**Reply by quantumquirkxyz** [reply]
https://learn.microsoft.com/es-es/azure/devops/repos/git/set-git-repository-permissions?view=azure-devops

https://fastercapital.com/es/palabra-clave/desarrolladores-deberían-acceso-repositorio-código-fuente.html
